kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -211,8 +210,8 @@
   .folder-header-text {
     display: inline-block;
     padding: 0;
-    margin: 0;
-    font-size: 16px;
+    margin: 7px 0;
```

Comment:
Not familiar with designs but this looks like something where we could follow 8px increments guidelines?

Reply:
This is a somewhat brittle way to make the header bar heights consistent because it is tightly coupled to the font size, so any change there will break the height again.

A more robust fix would be to set a fixed height (e.g. `48px`) on `.header-bar`